data-raw/generate-comp-data.R

```diff
@@ -19,8 +19,8 @@ hector_version <- packageVersion("hector")
 hector_commit <- system("git rev-parse --short HEAD", intern = TRUE)
 
 
-vars_to_save <- c(ATMOSPHERIC_CO2(), GLOBAL_TEMP(), RF_TOTAL(), RF_CO2(), HEAT_FLUX())
-dates_to_save <- 1750:2300
+vars_to_save <- c(ATMOSPHERIC_CO2(), GLOBAL_TEMP(), RF_TOTAL(), RF_CO2(), HEAT_FLUX(), OCEAN_C(), PH_HL())
+dates_to_save <- 1850:1950
```

Comment:
Why the date change?

Reply:
It is an arbitrary change but as we added more variables to the comparison data file I wanted to keep it smaller 🤷